superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/index.tsx

```diff
@@ -332,7 +332,7 @@ const time_grain_sqla: SharedControlConfig<'SelectControl'> = {
 
     const xAxis = controls?.x_axis;
     const xAxisValue = xAxis?.value;
-    if (xAxisValue === undefined || isAdhocColumn(xAxisValue)) {
+    if (isAdhocColumn(xAxisValue)) {
```

Comment:
Just to double check, does this work with the FF disabled?

Reply:
Yes, it does. Line 329 returns true when FF is disabled.

Reply:
Oh right, thanks for clarifying!